releng/com.espressif.idf.target/com.espressif.idf.target.target

```diff
@@ -122,5 +122,12 @@
 			<unit id="org.eclipse.embedcdt.feature.group" version="6.4.0.202307251916"/>
 			<unit id="org.eclipse.embedcdt.packs.feature.group" version="6.4.0.202307251916"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/linuxtools/update"/>
+			<unit id="org.eclipse.linuxtools.dataviewers.feature.feature.group" version="8.11.0.202306052033"/>
+			<unit id="org.eclipse.linuxtools.gprof.feature.feature.group" version="8.11.0.202306052033"/>
+			<unit id="org.eclipse.linuxtools.profiling.feature.group" version="8.11.0.202306052033"/>
+			<unit id="org.eclipse.linuxtools.profiling.remote.feature.group" version="8.11.0.202306052033"/>
```

Comment:
Hi @alirana01 Do we need all the features from the linuxtools? I thought we only need `org.eclipse.linuxtools.gcov-feature`? Also avoid mentioning a specific version so that it will continue to work with the latest versions when it's available.